### ISSUE TITLE
Cards fall off the then screen

### ISSUE THREAD

**Issue by magtimmermans** [athombv/homey#864]
**What is your Homey version (Settings → System)?**

0.9.3

**What did you try to achieve?**

Create a number of "then" actions

*Please include detailed steps how we can reproduce the issue*

Make a lot of "then" actions that does not fit on the screen

**What was the result?**

The top item is not completely visible.

**What did you expect as result?**

By using the scroller get the top item fully visible.

![2016-09-18 16_20_30-](https://cloud.githubusercontent.com/assets/17949840/18616451/5102d8c0-7dbc-11e6-8a98-aa4c8ed13597.png)

**Comment by magtimmermans** [comment]
Can this be updated ASAP, cannot change/update large workflows.

**Comment by markaswift** [comment]
This should be an easy fix. Guys?

**Comment by Arjanvde** [comment]
The problem here is that it align's the cards from the middle once the flow is reopend. Setting a top border (flow's need to stay under this line) or letting flows be aligned from the top should fix this.

**Comment by robertraaijmakers** [comment]
As far as I can tell this problem is solved (in Chrome) when doing the following. This will auto add a margin to the cards when possible. But when there are to many cards in the column will remove the margin and thus the center aligning.

Adding classes to CSS:

````
.cards::before {
    content: '';
    margin: auto;
}

.cards::after {
    content: '';
    margin: auto;
}
````

Remove the following:
`#editor>.column>.cards {
    justify-content: center;
}`

@WeeJeWel is there time to test this fix and implement it?

**Comment by robertraaijmakers** [comment]
Is there any update on the progress? It would be nice if I wouldn't lose my cards in the ..and.. and ..then columns. Maybe you have your reasons for it, if so, can you please let us know so we know this post is read?

Tested this fix on Chrome for Windows, Chrome for Mac OS X and Safari for Mac OS X.

Remove one line from CSS
`#editor>.column>.cards { justify-content: center; }`

Add one three lines of CSS
```
#editor>.column>.cards::before, #editor>.column>.cards::after {
   content: '';
   margin: auto;
}
```

**Comment by WeeJeWel** [comment]
That is a very nice fix! I've implemented it in 0.10.6, thanks!!
